src/flask_cognito_lib/services/cognito_svc.py

```diff
@@ -96,6 +97,116 @@ def exchange_code_for_token(
             "code_verifier": code_verifier,
         }
 
+        return self._request_token(data)
+
+    def refresh_token(
+        self,
+        refresh_token: str,
+    ) -> CognitoTokenResponse:
+        """
+        Exchange a refresh token for a new set of tokens
+
+        Parameters:
+        -----------
+        refresh_token : str
+            The refresh token to exchange for a new set of tokens
+
+        Returns:
+        --------
+        CognitoTokenResponse
+            A dataclass that holds the token response from Cognito
+
+        Raises:
+        -------
+        CognitoError
+            If the request to the endpoint fails
+            If the endpoint returns an error code
+        """
+        data = {
+            "grant_type": "refresh_token",
+            "client_id": self.cfg.user_pool_client_id,
+            "refresh_token": refresh_token,
+        }
+
+        return self._request_token(data)
+
+    def revoke_refresh_token(
+        self,
+        refresh_token: str,
+    ) -> None:
+        """
+        Revoke a refresh token
+
+        Parameters:
+        -----------
+        refresh_token : str
+            The refresh token to revoke
+
+        Raises:
+        -------
+        @see https://docs.aws.amazon.com/cognito/latest/developerguide/revocation-endpoint.html#revoke-sample-response
+
+        CognitoError
+            If the token isn't present in the request or if the feature is disabled for the app client
+            If the token that Amazon Cognito sent in the revocation request isn't a refresh token
+            If the client credentials aren't valid
+        """  # noqa
```

Comment:
What is the noqa for? I assume it's line length because of the URL?
I think we can probably remove the URL from the docstring which should negate the need for this noqa. Or alternatively, if we still need a noqa, it's good practice to annotate the specific reason, for example: `# noqa: E501`